sdk/nodejs/cmd/pulumi-language-nodejs/testdata/tsc/projects/l1-config-types/package.json

```diff
@@ -0,0 +1,10 @@
+{
+	"name": "l1-config-types",
+	"devDependencies": {
+		"@types/node": "^18"
+	},
+	"dependencies": {
+		"typescript": "^4.0.0",
+		"@pulumi/pulumi": "ROOT/artifacts/pulumi-pulumi-CORE.VERSION.tgz"
+	}
+}
```

Comment:
We should fix codegen to emit these missing newlines at the end of a file at some point :thinking: